src/Library/demos/File Dialog/main.blp

```diff
@@ -0,0 +1,35 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "File Dialog";
+  description: _("It collects the arguments that are needed to present a file chooser dialog to the user");
```

Comment:
This is not in line with the other descriptions. Please try to come up with description that are more inline with the other ones.

```suggestion
  description: _("Present a dialog to open and save files");
```